<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathForm</class>
 <widget class="QWidget" name="PathForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>251</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>251</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройка путей</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="dir_label_2">
         <property name="minimumSize">
          <size>
           <width>384</width>
           <height>13</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Локальная папка (КЭШ)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_1" stretch="0,0">
         <item>
          <widget class="QLineEdit" name="dir_line_edit">
           <property name="TypeField" stdset="0">
            <string>cash_path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="dir_button_dial">
           <property name="text">
            <string>Обзор</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="ini_label_2">
           <property name="text">
            <string>Путь к .ini файлу программы</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLineEdit" name="ini_line_edit">
             <property name="TypeField" stdset="0">
              <string>ini_path</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ini_button_dial">
             <property name="text">
              <string>Обзор</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="1">
          <widget class="QLabel" name="exe_label_2">
           <property name="text">
            <string>Путь к .exe файлу программы</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLineEdit" name="exe_line_edit">
             <property name="TypeField" stdset="0">
              <string>exe_path</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="exe_button_dial">
             <property name="text">
              <string>Обзор</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="1">
          <widget class="QLabel" name="exe_label_3">
           <property name="text">
            <string>Путь к datacnv</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLineEdit" name="datacnv_line_edit">
             <property name="TypeField" stdset="0">
              <string>cnv_path</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="datacnv_button_dial">
             <property name="text">
              <string>Обзор</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
